src/app/features/collectibles/components/collectible-hover.tsx

```diff
@@ -38,34 +38,19 @@ export function CollectibleHover({
         {collectibleTypeIcon}
       </Box>
       {onClickCallToAction && (
-        <IconButton
-          _focus={{ outline: 'focus' }}
-          alignItems="center"
-          bg="ink.background-primary"
-          borderRadius="50%"
-          display="flex"
-          height="30px"
-          justifyContent="center"
-          icon={
-            <ArrowUpIcon
-              transform="rotate(45deg)"
-              borderRadius="50%"
-              width="30px"
-              height="30px"
-              _hover={{ bg: 'ink.component-background-hover' }}
-            />
-          }
-          onClick={e => {
-            e.stopPropagation();
-            onClickCallToAction();
-          }}
-          position="absolute"
-          right="12px"
-          top="12px"
-          type="button"
-          width="30px"
-          zIndex={999}
-        />
+        <Box bg="ink.background-primary" position="absolute" right="12px" top="12px" zIndex={999}>
```

Comment:
Thanks @fbwoolf :clap: 

I tested this and its working perfectly. I'm not sure how I didn't get this myself, maybe as I was using `div` as a wrapper and not `Box`?

Anyway, glad you could solve it. Thanks!